libctru/include/3ds/services/ac.h

```diff
@@ -90,3 +90,36 @@ Result ACU_GetLastErrorCode(u32* errorCode);
  * @param errorCode Pointer to output the error code to.
  */
 Result ACU_GetLastDetailErrorCode(u32* errorCode);
+
+/**
+ * @brief Prepares a buffer to hold the configuration data to start a connection.
+ * @param config Pointer to a buffer of size at least 0x200 to contain the data.
+ */
+Result ACU_CreateDefaultConfig(void* config);
```

Comment:
Instead of a void* that's at least 0x200 bytes, could we get a struct (maybe named ACUConfig_t) that just has a char[0x200] in it?

Reply:
Something like this would work

```c
typedef struct
{
    u8 reserved[0x200];
} AcuConfig;
```